### PR TITLE
Update MOM calls for pika>=1.0

### PR DESCRIPTION
**Pull request by SukiCZ**

### CODE REVIEW

File: asab/mom/amqp/subscription.py [hunk 1]
```diff
@@ -47,31 +54,45 @@ def __init__(self, broker, exchange_name:str, exchange:bool=True, routing_key:st
...
+					self.Channel.queue_bind(
+						queue=self.QueueName,
+						exchange=self.ExchangeName,
+						routing_key=rk
+					)
+
+			def on_qos_applied(method):
+				self.Channel.basic_consume(self.QueueName, on_consume_message)
+
+		else:  # pika <= 13.0
```
Comment (ateska) [inline asab/mom/amqp/subscription.py]: `<1.0.0.0` in the comment I guess

**Comment by ateska** [comment]
1 smaller comment, otherwise mergeable.

File: asab/mom/amqp/subscription.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def on_channel_open(channel):
 			channel.basic_qos(callback=on_qos_applied, prefetch_count=int(self.Broker.Config['prefetch_count']))
 
 		def on_qos_applied(method):
-			if parse_version(pika.__version__) >= parse_version('1.0.0'):
+			if parse_version(pika.__version__) > parse_version('0.13.1'):
```
Comment (ateska) [inline asab/mom/amqp/subscription.py]: Nope, the correct test is `< 1.0.0`

File: asab/mom/amqp/subscription.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ def on_channel_open(channel):
 				callback=on_queue_declared,
 			)
 
-		if parse_version(pika.__version__) >= parse_version('1.0.0'):
+		if parse_version(pika.__version__) > parse_version('0.13.1'):
```
Comment (ateska) [inline asab/mom/amqp/subscription.py]: dtto

File: asab/mom/amqp/subscription.py (hunk 2 above)
Comment (SukiCZ) [reply asab/mom/amqp/subscription.py]: There are versions of pika `1.0.0b1` and `1.0.0b2` which are True for `< 1.0.0`. Maybe `< 1.a` ?

Comment (ateska) [reply asab/mom/amqp/subscription.py]: yes

**Comment by SukiCZ** [comment]
@ateska The test were changed in rev:0802a2e1 to `> parse_version('1.0.a')` as agreed. What am I missing?

**Pull request merged**